README.md

```diff
@@ -13,6 +13,8 @@ On Linux, it typically starts with `sysdefault` or `usb`.
 
 ![Recoding configuration panel UI](docs/panel.png)
 
+Note: For Audio input detection to work on linux, you need to have `arecord` available (see [Requirements](#requirements)).
```

Comment:
Could you please remove this line?
The less text the better, and I don't this line adds info.

Reply:
Done